spec/order_spec.rb

```diff
@@ -0,0 +1,39 @@
+require './lib/order.rb'
+
+describe Order do
+  let(:basket) { { :Burger => 2 } }
+  let(:menu) { {:Burger => 2} }
+  subject(:subject) {Order.new(menu)}
+
+
+  describe "#initialization" do
+
+    it "is initialized with an empty basket" do
+      expect(subject.basket).to be_empty
+    end
+
+    it {is_expected.to respond_to(:add_dish).with(2).arguments}
+  end
+
+  describe "#add order" do
+
+    it "adds selected dishes and quantities in basket" do
+      expect(subject.add_dish("Burger",2)).to eq "2x Burger(s) added to your basket"
```

Comment:
Metrics/LineLength: Line is too long. [84/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)